R/filterReads.R

```diff
@@ -215,6 +235,24 @@ filterReads <- function(se, assayName = "mod_prob",
                                 "AlignedFraction"] <- TRUE
         }
 
+        ## Region coverage fraction
+        if (!is.null(region) && minCoveredFraction > 0) {
+            tmp <- nnawhich(assay(se, assayName)[[nm]], arr.ind = TRUE)
+            tmpp <- split(pos(rowRanges(se))[tmp[, 1]], tmp[, 2])
+            tmpgr <- GRanges(
+                seqnames = seqnames(rowRanges(se))[1],
```

Comment:
`readModBam()` supports multiple regions on possibly different chromosomes - unless I'm misreading I think this may not do the intended thing if there are reads from multiple chromosomes in the object.